.github/workflows/build-test.yml

```diff
@@ -49,6 +49,8 @@ jobs:
                 PARAMETERS_SECRET: ${{ secrets.PARAMETERS_SECRET }}
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
               run: ./ci/test.sh
+            - name: Upload coverage to Codecov
+              uses: codecov/codecov-action@v3
```

Comment:
Yesterday at we talked with @sfc-gh-igarish that codecov has some problems with rate limitting on GH and we should use code cov token

Reply:
@sfc-gh-dprzybysz where does it read `${{ secrets.CODE_COV_UPLOAD_TOKEN }}` from? I added it to the build file but in the log it says no token was found although I can see `token=****` in the upload url.

Reply:
@sfc-gh-ext-simba-lb I just put the token for the GO driver. try it now. Yesterday I did it for Node.js.

Reply:
It works now thanks.